appcelerator/titanium_mobile
# fix(android): Fixed bug where setting SearchBar "hintText" after window open crashes as of 7.0.0

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-25832

**Test:**
1. Build and run the below code on Android.
2. Verify that the app starts up without crashing.
3. Verify that the top `SearchBar` reads "Hint Text".

```javascript
var window = Ti.UI.createWindow();
var searchBar = Ti.UI.createSearchBar({
	top: 0,
	width: Ti.UI.FILL,
	height: "50dp",
});
window.add(searchBar);
window.add(Ti.UI.createLabel({ text: "SearchBar Test" }));
window.addEventListener("open", function(e) {
	searchBar.hintText = "Hint Text";
});
window.open();
```
